### PR TITLE
Fix: set explicit version for ember-get-config

### PR DESCRIPTION
**Pull request by wozny1989**
After today's release `ember-get-config` [v1.0.0](https://github.com/mansona/ember-get-config/releases/tag/v1.0.0) in my case this non-set version in `package.json` crash app:
```
Build Error (broccoli-persistent-filter:Babel > [Babel: @embroider/macros]) in @embroider/macros/runtime.js

/Users/xxx/Documents/Projects/xxx/app/@embroider/macros/runtime.js: @babel/template placeholder "PATCH": Expected string substitution

Stack Trace and Error Report: /var/folders/q6/w2pl66k56ldbzpcmslws5vmh0000gn/T/error.dump.0db04ef46d623dc3645b2718a83e0ec1.log
```

Currently fetch the latest version of `ember-get-config` - 1.0.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "chalk": "^1.1.3",
     "ember-cli-babel": "^7.1.2",
     "ember-cli-import-polyfill": "^0.2.0",
-    "ember-get-config": "",
+    "ember-get-config": "^0.5.0",
```
Comment (jrjohnson) [inline package.json]: Can you set this to `^1.0.0`?

**Review by jrjohnson — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (wozny1989) [reply package.json]: With `1.0.0` project can't build with for e.g. package ~~[ember-pikaday](https://github.com/adopted-ember-addons/ember-pikaday)~~ `ember-cli-mirage` - have `ember-cli-moment-shim` in dependency. 

I see in 1.0.0 there are some `embroider` setup, maybe it's not safe for lower ember version(?). That's why I set previous working version

Comment (jrjohnson) [reply package.json]: It should be backwards compatible. I think anything that can't build with 1.0 should be considered a bug in that repo or a bug in get-config. Pinning this library to something else will cause more problems than it solves.

Comment (wozny1989) [reply package.json]: Understand 👍 I re-checked in fresh ember app and I see `ember-cli-mirage` have a problem with this error, not `ember-pikaday`.

I can change version to `^1.0.0` or close this PR. Just let me know

Comment (jrjohnson) [reply package.json]: I'd love to merge with `^1.0.0`. I think that's better than the current blank and probably stable for a very long time (hopefully longer than this addon needs to exist).

Comment (wozny1989) [reply package.json]: Sure, the version has been changed, thanks

**Pull request merged**